bin/charliecloud.h

```diff
@@ -90,6 +90,7 @@ extern int verbose;
 
 void containerize(struct container *c);
 void msg(int level, char *file, int line, int errno_, char *fmt, ...);
+unsigned long path_mount_flags(char *path);
```

Comment:
If we're going to use stuff from `charliecloud.c`, which it looks like we were already doing, I think we should use everything available. I believe there's a lot of duplication in `ch-checkns.c`.